app/src/main/java/nerd/tuxmobil/fahrplan/congress/base/LecturesAdapter.java

```diff
@@ -16,7 +16,7 @@
 import java.util.List;
```

Comment:
# Original commit message

> DateHelper#getDayChange has required a falllback of 600 seconds.
> This is fallback not required anymore, since a) its replacement method DateParser#getDateTime
> is more robust and b) this default value is already provided in ParserTask#parseFahrplan
> and is only required for none Pentabarf schedules.

# Corrected commit message

> DateHelper#getDayChange has required a fallback of 600 minutes.
> This fallback is not required anymore since its replacement method DateParser#getDateTime
> is more robust.
> Please note that the very same default value is still used in ParserTask#parseFahrplan
> because the corresponding <day_change> XML node is only present in the
> schedule.xml file when retrieved from a Pentabarf API.

Please replace the message part.